### ISSUE TITLE
Cannot parse  as CloudRegion

### ISSUE THREAD

**Issue by 0anton** [ergut/mcp-bigquery-server#4]
For every request, for example this:
```
{
  `sql`: `
SELECT schema_name
FROM INFORMATION_SCHEMA.SCHEMATA
ORDER BY schema_name;
`
}
```

I'm getting an error:

```Error executing code: MCP error 400: Cannot parse  as CloudRegion.```

There is a double space after the word "parse". I don't see the word _CloudRegion_ in the source code of this module.

I've tried both _project id_ and _project number_ - same error.

`bq` cli can list table with out any problems.

**Comment by silvanm** [comment]
In my case, the problem was the multi-region setup of my DB. I modified the code a bit to accommodate for this.

https://github.com/ergut/mcp-bigquery-server/pull/5

**Comment by 0anton** [comment]
Thanks für PR, @silvanm! The fixed list of regions is a bit risky (not „timeless“), as Google adds regions from time to time, but it is easy to correct. I like the idea with default region „EU“, too 🙂, but sure our US friends will disagree with its commonality.

I‘ve noticed the BigQuery MCPs all require the region specified. It‘s very annoying if you have tables in mixed regions (e.g. „EU“ and „europe-west1“), as it randomly fails as traffic moves from table to table.

If I run „bq query“ from CLI I never have to bother about region. It just runs, as soon as I have permissions to the object. I wonder if MCP can also be made region agnostic….

I also don‘t remember if BigQuery API requires regions specification.

**Comment by ngohuuduc** [comment]
@0anton  Do you still face the same issue ? I'm facing this issue and not able to solve.

**Comment by 0anton** [comment]
@ngohuuduc yes

**Comment by ngohuuduc** [comment]
> [@ngohuuduc](https://github.com/ngohuuduc) yes

seems like the issue is actually coming from API itself ?

**Comment by ngohuuduc** [comment]
@0anton  : you may try my fork version
https://github.com/ngohuuduc/mcp-bigquery-server 

it's just work for me.

**Comment by ngohuuduc** [comment]
@0anton Can you try again with the latest release ? Then we can close this issue.

**Comment by 0anton** [comment]
Yes, I think this current state is the best possible. BigQuery forces to specify the region (probably to route traffic to the correct pool of services), so quite region-independent behaviour is not quite reachable. Thanks for the change @ngohuuduc and closing…
